sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryProperties.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+import java.util.Map;
+
+/**
+ * Represents properties about a {@link Geometry}.
+ */
+public final class GeometryProperties {
+    private final GeometryBoundingBox boundingBox;
+    private final Map<String, Object> additionalProperties;
+
+    /**
+     * Constructs an empty properties.
+     */
+    public GeometryProperties() {
+        this(null, null);
+    }
```

Comment:
Is there a need for this ctor to be public - it seems like no sane user would need to call this? Also, should there be a ctor that takes just the bounding box?

Reply:
It is better to use setter pattern in this class. Search is only asking additional properties.

Reply:
I'm thinking of removing this class and using its components directly in `Geometry`.

Reply:
Removed `GeometryProperties`.